### PR TITLE
Feat/action button

### PR DESCRIPTION
**Pull request by maimonav**

### CODE REVIEW

File: packages/plugin-button/web/src/typeMapper.js [hunk 1]
```diff
@@ -1,9 +1,13 @@
-import { BUTTON_TYPE } from './constants';
+import { LINK_BUTTON_TYPE, ACTION_BUTTON_TYPE } from './constants';
 import { alignmentClassName, sizeClassName } from './classNameStrategies';
 import ButtonViewer from './components/button-component';
 
 export const typeMapper = () => ({
-  [BUTTON_TYPE]: {
+  [LINK_BUTTON_TYPE]: {
+    component: ButtonViewer,
```
Comment (shaulgo) [inline packages/plugin-button/web/src/typeMapper.js:7]: duplicate of below

File: examples/main/shared/viewer/ViewerPlugins.jsx [hunk 2]
```diff
@@ -1,5 +1,8 @@
 import theme from '../theme/theme';
-import { BUTTON_TYPE } from 'wix-rich-content-plugin-button/dist/module.viewer';
+import {
+  LINK_BUTTON_TYPE,
```
[the remark this answers is not in the document]
Comment (maimonav) [reply examples/main/shared/viewer/ViewerPlugins.jsx:3]: fixed.

File: examples/main/shared/editor/EditorPlugins.jsx [hunk 3]
```diff
@@ -528,49 +564,10 @@ const config = {
...
-    },
-    onTextColorAdded: color => (userButtonTextColors = [color, ...userButtonTextColors]),
-    onBackgroundColorAdded: color =>
-      (userButtonBackgroundColors = [color, ...userButtonBackgroundColors]),
-    onBorderColorAdded: color => (userButtonBorderColors = [color, ...userButtonBorderColors]),
-    getTextColors: () => userButtonTextColors,
-    getBorderColors: () => userButtonBorderColors,
-    getBackgroundColors: () => userButtonBackgroundColors,
-  },
-  [TEXT_HIGHLIGHT_TYPE]: {
```
[the remark this answers is not in the document]
Comment (maimonav) [reply examples/main/shared/editor/EditorPlugins.jsx:563]: fixed.

File: packages/common/web/statics/locale/messages_en.json [hunk 4]
```diff
@@ -187,6 +187,7 @@
   "GallerySettings_Image_Ratio": "Image ratio",
   "GallerySettings_Thumbnail_Placement": "Thumbnail placement",
   "ButtonPlugin_InsertButton_Tooltip": "Add a button",
+  "ActionButtonPlugin_InsertButton_Tooltip": "Add an action-on-click",
```
[the remark this answers is not in the document]
Comment (maimonav) [reply packages/common/web/statics/locale/messages_en.json:190]: fixed.

File: packages/plugin-button/web/src/typeMapper.js (hunk 1 above)
Comment (maimonav) [reply packages/plugin-button/web/src/typeMapper.js:7]: fixed.

File: packages/plugin-button/web/src/toolbar/insert-buttons.jsx [hunk 5]
```diff
@@ -1,15 +1,19 @@
...
   const rel = settings?.relValue === '_nofollow';
   const target = settings?.anchorTarget ? settings?.anchorTarget === '_blank' : true;
   return [
     {
       name: 'Button',
-      tooltipText: t('ButtonPlugin_InsertButton_Tooltip'),
+      tooltipText:
+        LINK_BUTTON_TYPE === type
+          ? t('LinkButtonPlugin_InsertButton_Tooltip')
+          : t('ActionButtonPlugin_InsertButton_Tooltip'),
```
[the remark this answers is not in the document]
Comment (maimonav) [reply packages/plugin-button/web/src/toolbar/insert-buttons.jsx:16]: fixed.

**Pull request merged**